nms/app/packages/magmalte/grafana/dashboards/CWFDashboards.js

```diff
@@ -100,8 +100,45 @@ export const CWFSubscriberDBData: GrafanaDBData = {
           description:
             "Should just be 0 for subscribers without an active session and 1 for those with. Not sure what's up with this now.",
         },
+        {
+          title: 'User Growth',
+          targets: [
+            {
+              expr: 'count(sum(session_start{msisdn=~".*"}) by (msisdn))',
```

Comment:
Also what does msisdn matching .* do ? is this right ? Should it be $msisdn instead. @Scott8440 am i missing something here. Thanks

Reply:
Actually I was planning on adding alerts to this metric. But Grafana does not allow a template variable in the metric if we have to configure alerts. Hence i just used .*  to match any msisdn. This table should just indicate how many unique user sessions have been created over certain time. So one can compare if there are any new users added based on the given time series.   If a new unique session comes in, session_start metric will get incremented for it and hence it will have a new  range vector. The count() will thus increment by 1 indicating that there is a new user.

Reply:
You shouldn't be using Grafana for alerts anyways. The alertmanager integration we have in the NMS is a lot more powerful and reliable.

Reply:
Ok. I did not knew about this. Thanks for pointing out.